sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2.scala

```diff
@@ -67,6 +68,46 @@ object HiveThriftServer2 extends Logging {
     }
   }
 
+  def runScript(sqlContext: HiveContext) {
+    val hiveConf = sqlContext.hiveconf
+    val script = hiveConf.get("hive.server2.init.script", null)
```

Comment:
Is this a standard conf? Googling it didn't turn up anything.

Reply:
Do you resolve this HiveSQLException? I ran into the same exception.